Src/BootCamp.Chapter/Conversions.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Text.RegularExpressions;
+
+namespace BootCamp.Chapter
+{
+    class Conversions
+    {
+        public static void Demo()
+        {
+            Console.WriteLine("Convert Decimal To Binary");
+            ConvertDecimalToBinary();
+            Console.WriteLine();
+            Console.WriteLine("Convert Binary To Decimal");
+            ConvertBinaryToDecimal();
+        }
+
+        private static void ConvertDecimalToBinary()
```

Comment:
This function does more than 1 thing.
Split reading input and converting part.
You can still use both in one function, it should just be split in at least 2 parts.
And it is more natural that was, we convert something, so it makes sense that we pass input to convert it.